src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString
+    .replaceAll('\t', '')
+    .split(';')
+    .reduce((prev, element) => {
+      const elements = element.split(':');
+      let key = elements[0];
+      const value = elements[1];
+
+      key = key.replaceAll(' ', '').replaceAll('\n', '');
+
+      if (key.length === 0) {
+        return prev;
+      }
+
+      prev[key] = value.trim();
+
+      return prev;
+    }, {});
```

Comment:
The variable `sourceArray` is misleading because the final result is actually an object, not an array. Consider renaming it to better reflect its purpose, such as `styleObject` or `cssStyles` to improve code readability. [CODE STYLE]